R/teal_slices.R

```diff
@@ -44,7 +44,7 @@
 #' }
 #'
 #' @export
-teal_filters <- function(...,
+teal_slices <- function(...,
```

Comment:
Are we OK with having two functions of the same name in different packages? None of them `Depend` on the other so masking will not be entirely predictable.

Reply:
I think moving `teal.slice` to depends solves the problem as `teal.slice` will be loaded always before and thus `teal_slices` will be masked with R throwing message.

Reply:
`Depends` and masking will solve the problem of usage, which is good enough. The only drawback is that when one types `?teal_slices` in the console, they will have to pick one of two topics.

I will try to come up with a different name for the function here.

Reply:
Great! :ok_hand: